controls/radspreadsheet/features/ui-headers-and-footers.md

```diff
@@ -0,0 +1,94 @@
+
+
+* **Scale with document**: Indicates whether headers and footers should be scaled along with the document when being rendered. When this option is disabled, headers and footers are always rendered at 100% scale.
+
+
+* **Align with page margins**: Indicates whether headers and footers should align with the left and right margins of the document. When this setting is disabled, headers and footers are aligned with the default left and right margins.
+
+## Custom Header/Footer
+
+The content of the header/footer is customizable and you can choose the text that will be included. This could be done through the *Custom Header...* and *Custom Footer...* buttons in the Page Setup dialog. They will open a new dialog that gives you the ability to insert the desired content.
+
+
+* **Left section**: Represents the content of the left header/footer section. The content of this section is flowing from the left to the right page margin with left horizontal alignment.
+
+* **Center section**: Represents the content of the center header/footer section. The content of this section is flowing from the left to the right page margin with horizontal alignment center.
+
+* **Right section**: Represents the content of the right header/footer section. The content of this section is flowing from the left to the right page margin with right horizontal alignment.
+
+
+Have in mind that the amount of content that can be inserted the three sections of a header/footer is limited and depends on the available space for rendering them. In case you exceed this limitation, a dialog will appear to notify that you will need to decrease the inserted content.
+
+#### **Figure 4: Header dialog**
+![](images/RadSpreadsheet_UI_Headers_Footers_03.png)
+
+Along with the text in the header/footer sections, you can insert several fields that will be evaluated depending on the current context of the header/footer. The available options are shown in **Figure 4**.
+
+1. **Insert Page Number**: Allows you to insert a page number field. This field is evaluated for each page and is rendered as the current page number value.
+
+2. **Insert Number of Pages**: Allows you to insert a number of pages field. This field is evaluated and displayed as the total number of pages.
+
+3. **Insert Date**: Allows you to insert a date field. This field is evaluated as the date on which the printing or PDF export has started.
+
+4. **Insert Time**: Allows you to insert a time field. This field is evaluated as the time when the printing or PDF export has started.
+
+5. **Insert File Path**: Allows you to insert the file folder field. This field is evaluated as the full folder path to the currently opened file. Note that when exporting a Workbook, this field is evaluated as empty string because the Workbook instance is not related to any concrete file in the file system.
```

Comment:
Does this mean you can set the file path and it will be only respected when printing?